projects/js-packages/eslint-changed/src/cli.js

```diff
@@ -140,8 +148,32 @@ async function main( process, argv, program ) {
 		return res.stdout;
 	}
 
-	const eslint = new ESLint();
 	debug( 'Using ESLint version', ESLint.version );
+
+	// Parse --eslint-options.
+	const eslintOptions = {};
+	if ( argv.eslintOptions ) {
+		while ( argv.eslintOptions.length > 0 ) {
+			let k = argv.eslintOptions.shift(),
+				v;
+			const i = k.indexOf( '=' );
+			if ( i >= 0 ) {
+				v = k.substring( i + 1 );
+				k = k.substring( 0, i );
+			} else {
+				v = argv.eslintOptions.shift();
```

Comment:
This works, though I personally prefer a stricter structure (e.g. only allow one of `k=v` or `k v`) as opposed to "smart" parsing.